handlers.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"fmt"
+	"github.com/gorilla/mux"
+	log "github.com/Sirupsen/logrus"
+	tid "github.com/Financial-Times/transactionid-utils-go"
+)
+
+const mapiPath = "/eom-file/"
+const matPath ="/content-transform/"
+
+var logger = log.New()
+
+type Handlers struct {
+	mapiAuth      string
+	matHostHeader string
+	mapiUri       string
+	matUri        string
+}
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "pong")
+}
+
+func (h Handlers) buildInfoHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "build-info")
+}
+
+func (h Handlers) contentPreviewHandler(w http.ResponseWriter, r *http.Request) {
+	transactionId := tid.GetTransactionIDFromRequest(r)
+	logger.Formatter = new(log.JSONFormatter)
+	logger.WithFields(log.Fields{
+		"requestUri" : r.RequestURI,
+		"transactionId" : transactionId,
+	}).Info("request started");
+
+	vars := mux.Vars(r)
+	uuid := vars["uuid"]
+
+	success, mapiResp := h.getNativeContent(uuid, transactionId, w)
+	if !success { return }
+	success, matResp := h.getTransformedContent(uuid, *mapiResp, w)
+	if(!success) {
+		mapiResp.Body.Close()
+		return
+	}
+	w.Header().Set(tid.TransactionIDHeader, matResp.Header.Get(tid.TransactionIDHeader))
+	io.Copy(w, matResp.Body)
+	matResp.Body.Close()
+}
+
+func ( h Handlers) getNativeContent(uuid string, transactionId string, w http.ResponseWriter) (ok bool, resp *http.Response) {
+	logger.Formatter = new(log.JSONFormatter)
```

Comment:
Go provides support for passing around variables associated with a request lifecycle via the context package, and 
recommended best practice for using this is to specify a context as the first argument to your function (see the [transaction id utils library](https://github.com/Financial-Times/transactionid-utils-go) for more info). Do you think we should use this here?